frontend/src/components/Banner/Banner.tsx

```diff
@@ -16,7 +14,7 @@ import { BxsErrorCircle, BxsInfoCircle } from '~/assets/icons'
 
 import { BannerVariant } from '~theme/components/Banner'
 
-import Link from '../Link'
+import { useMdComponents } from '../../hooks/useMdComponents'
```

Comment:
nit: should we use the alias path instead?

```suggestion
import { useMdComponents } from '~hooks/useMdComponents'
```